src/resolve/borders.ts

```diff
@@ -35,8 +35,22 @@ export function border(value: string, theme?: TwTheme): StyleIR | null {
       colorType = `borderRight`;
       break;
   }
+  const hasBorderColor = color(colorType, rest, theme?.borderColor);
+  if (hasBorderColor) {
+    return color(colorType, rest, theme?.borderColor);
+  }
 
-  return color(colorType, rest, theme?.borderColor);
+  // Finally Handle Arbitrary Case
```

Comment:
very small nitpick, but lets modify the comment to clarify that this is handling arbitrary _width_ case (because arbitrary colors are already handled above)

Reply:
Changint the comment to 
```
  // Finally Handling Arbitrary Width Case
  ```